radio/src/functions.cpp

```diff
@@ -376,9 +376,7 @@ void evalFunctions()
           case FUNC_SET_TIMER:
             timerSet(CFN_TIMER_INDEX(cfn), CFN_PARAM(cfn));
             break;
-#endif
```

Comment:
It was better before (each case surrounded by its #ifdef if needed, no nested #ifdef), no?